src/views/operations/useGetOperations.tsx

```diff
@@ -162,40 +158,14 @@ const fetchOperationsAndUpdateTokensInfo = async (
 
 // when we fetch token transfers related to our accounts we might fetch
 // the ones which are initiated by us and hence we get duplicates
-const LOOK_AHEAD = 10; // this should be enough to cover all the cases and not cause O(N^2) lookups
 export const filterDuplicatedTokenTransfers = (
   operations: TzktCombinedOperation[]
 ): TzktCombinedOperation[] => {
-  const result: TzktCombinedOperation[] = [];
-
-  for (let i = 0; i < operations.length; i++) {
-    const operation = operations[i];
-    if (operation.type !== "token_transfer") {
-      result.push(operation);
-      continue;
-    }
-
-    // if token transfer was initiated by a migration or origination then it won't have a duplicate record
-    if (operation.transactionId === undefined) {
-      result.push(operation);
-      continue;
-    }
-
-    let hasDuplicate = false;
-    for (
-      let j = max([i - LOOK_AHEAD, 0]) as number;
-      j < (min([i + LOOK_AHEAD, operations.length]) as number);
-      j++
-    ) {
-      if (operations[j].id === operation.transactionId) {
-        hasDuplicate = true;
-        break;
-      }
-    }
-    if (!hasDuplicate) {
-      result.push(operation);
-    }
-  }
+  const transactionIds = new Set(
+    operations.filter(op => op.type !== "token_transfer").map(op => op.id)
```

Comment:
because if I include token transfers and then skip all token transfers with such an id then it'll erase all token transfers (including those without duplicates in transactions)

Reply:
Actually, that's not the reason. It's in the select function of the useInfiniteQuery params. It must always call this function